src/routes/about.lazy.tsx

```diff
@@ -0,0 +1,55 @@
+import { ColorContext } from "@/color-context";
+import { ExpressionCount } from "@/components/expression-counter";
+import { createLazyFileRoute, Link } from "@tanstack/react-router";
+import { useContext } from "react";
+
+export const Route = createLazyFileRoute("/about")({
+  component: About,
+});
+
+function About() {
+  const { color } = useContext(ColorContext);
+  return (
+    <div
+      className="page-padding min-h-screen w-screen flex items-center justify-center"
+      style={{ backgroundColor: color }}
+    >
+      <div className="flex flex-col gap-6">
+        <h1 className="font-logo">Om Prosjektet</h1>
+        <p>
+          Det var en kald vår. Året var 2024 og to informatikk studenter med en
+          sær humor basert på spesielle ordtak, utrykk og ord, får en visjon..
+          En visjon om å lage en nettside som samler alle disse "utrykkene" på
+          ett sted. Rett og slett en liten hyllest til det norske språk. Dagens
+          Ord var født. Konseptet "Dagens Ord" er veldig simpelt. Hver dag får
```

Comment:
We have to decide on Dagens Ord or Dagens ord😮‍💨

Reply:
I propose "Dagens Ord" because it is a "egennavn". It is common in cases such as "Det Norske Veritas"

Reply:
Agree!